.github/workflows/release.yml

```diff
@@ -0,0 +1,105 @@
+name: Releases
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  build-windows:
+    runs-on: windows-latest
+    timeout-minutes: 60
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+      - uses: msys2/setup-msys2@v2
+        with:
+          update: true
+          path-type: inherit
+          install: >-
+            curl
+            make
+            cmake
+
+      - shell: msys2 {0}
+        run: |
+          pacman -S mingw-w64-x86_64-gcc mingw-w64-x86_64-cmake mingw-w64-x86_64-make mingw-w64-x86_64-curl libcurl openssl-devel libcurl-devel autoconf texinfo mingw-w64-x86_64-libtool --noconfirm
+          make
+
+          mkdir ./txiki-windows-x86
+          cp ./build/tjs.exe ./txiki-windows-x86/tjs.exe
+          cp ./build/libtjs.a ./txiki-windows-x86/libtjs.a
+          cd ./txiki-windows-x86
+          ldd ./tjs.exe | grep '=>' | awk '{print $3}' | xargs -I '{}' cp -v '{}' .
+
+      - name: Upload windows Artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: txiki-windows-x86
+          path: ./txiki-windows-x86
+
+  build-macos-arm64:
```

Comment:
Rather than making 2 builds, we can make a single "fat" one by calling cmake manually like so: `cmake -DCMAKE_OSX_ARCHITECTURES="x86_64;arm64" ..`

Reply:
I am not familiar with C project compilation, and I don't have a Mac for testing, so I don't know the output directory format.  T_T

Reply:
Look at his PR: https://github.com/saghul/txiki.js/pull/606 once it lands you can rebase yours and use `make BUILD_MACOS_MULTIARCH=1` for a "fat" build.